aaf_walking_group/action/WalkingGroup.action

```diff
@@ -0,0 +1,5 @@
+string group
+string[] parameters
```

Comment:
This is just me not really knowing how the gcal yaml architecture works. The thing is that the launch file has parameters, which could of course be fixed as defaults in the launch file, but if can use the yaml file to prepopulate the goal from that, I'd rather do it that way. Maybe we can have a chat on how you envisioned the use of the abstract task server?